README.md

```diff
@@ -315,30 +316,32 @@ compensation.
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
   may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
   under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
+  - ☝️ Vague, huh?
```

Comment:
To my taste, this crosses into the "too informal for this topic" range. It's not really vague, it's just informal—not a legal concept. How about updating to:

"Unlike some of the legal terms we discuss, the term *startup* is informal, and not everyone uses it consistently. As famous investor Paul Graham [emphasizes](http://www.paulgraham.com/growth.html), it's likely best to think of a startup as any [early stage](#stages-of-a-startup) company intending to grow quickly."

Reply:
I could not find my comments for a revised startup definition anywhere in commits, where I suggested that we used the Paul Graham quote... Any guesses where it ended up? (Also, this is WIP so hang tight before review. Ignore this PR for now.)

Reply:
corrected!